### DISCUSSION TITLE
[Marketplace] Search returns wrong results

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#385] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

12.0.1

### Description

Hi

 I'm searching for templates having "Single View" in their name; however, the result also shows the micro-service "Timeline Viewer", which doesn't have either _single_ or _view_ in it.

<img width="915" alt="image" src="https://github.com/mia-platform/community/assets/38134891/a49b73b1-3d0b-4863-8159-36b89a15b084">

Is this the expected behavior?

**Comment by fredmaggiowski** [comment, marked as answer]
Marketplace uses fuzzy search, timeline **view**er matches the query :)

**Reply by davidebianchi** [reply]
The filter returns the elements in order of grade of matching queries, so further down the list there are also some elements which are not perfect matching, but could also help someone which does not know the exact name of something but want to search for some keyword

**Reply by flower-of-the-bridges** [reply]
hi @davidebianchi @fredmaggiowski thanks for the explanation!
